src/alpine/3.10/helix/amd64/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7.3-alpine3.10
+FROM python:3.7.6-alpine3.10
```

Comment:
Is there any reason to not use the `3.7-alpine3.10` floating tag?

Reply:
No specific reason, was just using the opportunity to update to a compatible newer version.  Should I change it?